backend/app/rest/api/rest_test.go

```diff
@@ -330,7 +330,8 @@ func startupT(t *testing.T) (ts *httptest.Server, srv *Rest, teardown func()) {
 			WordPressImporter: &migrator.WordPress{DataStore: dataStore},
 			NativeImporter:    &migrator.Native{DataStore: dataStore},
 			NativeExporter:    &migrator.Native{DataStore: dataStore},
-			Cache:             &cache.Nop{},
+			Mapper:            &migrator.UrlMapper{},
+			Cache:             memCache,
```

Comment:
Is there a reason for adding memCache here instead of nop cache? It adds another element into the picture. I'm not saying this is the bad thing as rest test some kind of integration resting here, but what was your motivation? Is anything you tested didn't work without it?

Reply:
ok, makes sense